### ISSUE TITLE
Wrong type signatures in master branch

### ISSUE THREAD

**Issue by frogmaster** [exhuma/puresnmp#91]
## Issue Description
Type signatures are wrong on master branch and in 10.2-post1 release. They seem to be fine on develop. Seems to be some kind on branch merging problem...

puresnmp/api/raw.py|59 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/raw.py|110 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/raw.py|176 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/raw.py|282 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/raw.py|302 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/raw.py|497 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|127 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|152 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|178 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|209 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|224 col 1 error| Type signature has too few arguments  [syntax]
puresnmp/api/pythonic.py|264 col 1 error| Type signature has too few arguments  [syntax]

**Comment by exhuma** [comment]
Hi.

Thanks for reporting this. I will have a look. Was this detected using `mypy`?

As a small heads-up: I'm in the process of working on a major release which also includes support for SNMPv3. This will change the external API. The project is versioned using SemVer. If you want to be on the safe side, pin your dependencies to `<2.0` so you won't run into issues once it's released.

There's just a few things to verify before I release it so it should hit beta any time soon. It's been extensively tested as internal alpha already.

**Comment by frogmaster** [comment]
Yes, output from mypy. Looked at blame on master branch also, method def and type-hint comment are ages apart and the comment is missing the last argument. I guess it would be worth fixing for those who'll pin the dep. Good to know that a new API is coming, as I am writing a brand new app, guess I will concentrate on other aspects first and start using the new API asap.

**Comment by exhuma** [comment]
I might be able to squeeze an alpha release in here somehow so you can give it a try. Docstrings are extensive so you should be able to figure things out without external docs.

I'll definitely plan to tackle the type-hints in the 1.x branch anyway as well.
